### ISSUE TITLE
License

### ISSUE THREAD

**Issue by ieee8023** [lambert-x/medical_mae#4]
Hello,

Can you tell me what the license is for this code and model? I would like to add it to the [TorchXRayVision](https://github.com/mlmed/torchxrayvision) library which is under an Apache License.

-Joseph

**Comment by ameer-khu** [comment]
Hi Joseph, I liked the idea of hosting these MAEs under TorchXRayVision library because it is very convenient and easy to use the models hosted there, even on the HuggingFace. So, I noticed something which I want to share. Most of this repository files says following: 

"Copyright (c) Meta Platforms, Inc. and affiliates.
All rights reserved.

This source code is licensed under the license found in the
 LICENSE file in the root directory of this source tree.
 --------------------------------------------------------
 References:
 DeiT: https://github.com/facebookresearch/deit
 BEiT: https://github.com/microsoft/unilm/tree/master/beit
 --------------------------------------------------------"

I checked the above URL for LICENSE and found that either the license is Apache or MIT. I'm sharing license links below for your reference. I hope to see these MAEs model under TorchXRayVision library. 

Ref. links:
https://github.com/facebookresearch/deit/blob/main/LICENSE
https://github.com/microsoft/unilm/blob/master/LICENSE
